src/Razor/src/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -65,5 +65,7 @@
     <Asset Type="Microsoft.VisualStudio.MefComponent"  Path="Microsoft.VisualStudio.LiveShare.Razor.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly"  Path="Microsoft.VisualStudio.LanguageServerClient.Razor.dll" />
     <Asset Type="Microsoft.VisualStudio.MefComponent"  Path="Microsoft.VisualStudio.LanguageServerClient.Razor.dll" />
+    <Asset Type="Microsoft.VisualStudio.Assembly"  Path="Microsoft.VisualStudio.LanguageServer.ContainedLanguage.dll" />
```

Comment:
You'll also need to directly reference this project in the RazorExtension project like: https://github.com/dotnet/aspnetcore-tooling/blob/038d9df582689d9a1f20eabea1e6a8446e60ca66/src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj#L149-L158

Reply:
Thanks, added it.

---
In reply to: [437133106](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437133106) [](ancestors = 437133106)